packages/fxa-auth-server/lib/routes/session.js

```diff
@@ -373,16 +374,19 @@ module.exports = function(
         if (!account.primaryEmail.isVerified) {
           await signupUtils.verifyAccount(request, account, options);
```

Comment:
IIRC there is another push notification sent somewhere inside `verifyAccount` here, which could also trigger this issue.